src/DynamoCoreWpf/Interfaces/LayoutSpecification.cs

```diff
@@ -0,0 +1,253 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace Dynamo.Wpf.Interfaces
+{
+    /// <summary>
+    /// Represents the specification for the library items layout in library view.
+    /// </summary>
+    public class LayoutSpecification
+    {
+        /// <summary>
+        /// Default constructor
+        /// </summary>
+        public LayoutSpecification()
+        {
+            sections = new List<LayoutSection>();
+        }
+
+        /// <summary>
+        /// List of LayoutSections
+        /// </summary>
+        public List<LayoutSection> sections { get; set; }
```

Comment:
When this object is marshaled to JS then the property names starts with lowercase. I just wanted to keep it with the same convention, so confusion.